taskt/Core/Automation/Commands/IE Browser Commands/IEElementActionCommand.cs

```diff
@@ -1,53 +1,66 @@
+        [SampleUsage("")]
+        [Remarks("")]
+        public DataTable v_WebSearchParameter { get; set; }
+
         [XmlElement]
-        [Attributes.PropertyAttributes.PropertyDescription("IE Element Action")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Invoke Click")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Left Click")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Middle Click")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Right Click")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Get Text")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Set Text")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Get Attribute")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Set Attribute")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Fire onmousedown event")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Fire onmouseover event")]
-        [Attributes.PropertyAttributes.InputSpecification("Select the appropriate corresponding action to take once the element has been located")]
-        [Attributes.PropertyAttributes.SampleUsage("Select from **Invoke Click**, **Set Text**, **Get Text**, **Get Attribute**")]
-        [Attributes.PropertyAttributes.Remarks("Selecting this field changes the parameters that will be required in the next step")]
+        [PropertyDescription("IE Element Action")]
+        [PropertyUISelectionOption("Invoke Click")]
+        [PropertyUISelectionOption("Left Click")]
+        [PropertyUISelectionOption("Middle Click")]
+        [PropertyUISelectionOption("Right Click")]
+        [PropertyUISelectionOption("Get Text")]
+        [PropertyUISelectionOption("Set Text")]
+        [PropertyUISelectionOption("Get Attribute")]
+        [PropertyUISelectionOption("Set Attribute")]
+        [PropertyUISelectionOption("Fire onmousedown event")]
+        [PropertyUISelectionOption("Fire onmouseover event")]
+        [InputSpecification("Select the appropriate action to perform on an element once it has been located.")]
+        [SampleUsage("")]
+        [Remarks("Selecting this field changes the parameters that will be required in the next step.")]
         public string v_WebAction { get; set; }
+
         [XmlElement]
-        [Attributes.PropertyAttributes.PropertyDescription("Action Parameters")]
-        public System.Data.DataTable v_WebActionParameterTable { get; set; }
+        [PropertyDescription("Action Parameters")]
+        [InputSpecification("Enter the action parameter values based on the selection of an Element Action.")]
+        [SampleUsage("{vParameterValue}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public DataTable v_WebActionParameterTable { get; set; }
 
         [XmlIgnore]
         [NonSerialized]
```

Comment:
all these private variables should be underscore camel cased. I may have missed some of these in previous reviews...There's one in FindBrowser as well.

Reply:
done